epictrack-web/src/components/work/workStaff/WorkStaffList.tsx

```diff
@@ -41,13 +42,19 @@ const WorkStaffList = () => {
   wsData.forEach((value, index) => {
     if (value.staff.length > 0) {
       const roles = value.staff
+        .filter(
+          (p) =>
+            p.role.id !== WorkStaffRole.TEAM_LEAD &&
+            p.role.id !== WorkStaffRole.RESPONSIBLE_EPD

```

Comment:
You can use IN clause instead of &&
![WorkStaffRole.TEAM_LEAD, WorkStaffRole.RESPONSIBLE_EPD].includes(p.role.id)

Reply:
i update it